HEC.FDA.ViewModel/Hydraulics/GriddedData/GriddedImporterVM.cs

```diff
@@ -203,26 +223,30 @@ private void SaveNew()
         {
             string destinationDirectory = Connection.Instance.HydraulicsDirectory + "\\" + Name;
             Directory.CreateDirectory(destinationDirectory);
-            List<HydraulicProfile> pathProbs = new();
+            List<HydraulicProfile> hydraulicProfiles = new();
             foreach (WaterSurfaceElevationRowItemVM row in ListOfRows)
             {
-                string fileNameFromChildElementDir = getFilePathFromChildElement(row);
-                pathProbs.Add(
-                    new HydraulicProfile(row.Probability, fileNameFromChildElementDir, null));
-                StudyFilesManager.CopyDirectory(row.Path, row.Name, destinationDirectory);
+                string fileNameFromChildElementDir = GetFilePathFromChildElementRoot(row);
+                HydraulicProfile prof = new(row.Probability, fileNameFromChildElementDir, null);//profile name is not used for gridded data.
+                hydraulicProfiles.Add(prof);
+                string sourceDirectory = Directory.GetParent(row.Path).FullName;
+                StudyFilesManager.CopyDirectory(sourceDirectory, row.Name, destinationDirectory);
             }
 
             int id = GetElementID<HydraulicElement>();
-            HydraulicElement elementToSave = new(Name, Description, pathProbs, HydraulicDataSource.WSEGrid, id);
+            HydraulicElement elementToSave = new(Name, Description, hydraulicProfiles, HydraulicDataSource.WSEGrid, id);
             base.Save(elementToSave);
         }
 
-        private string getFilePathFromChildElement(WaterSurfaceElevationRowItemVM row)
+        /// <summary>
```

Comment:
This method was a mess. It should get the file path of the tif from child element root. None of the naming conventions conveyed that, and the method for actually gettting it was convoluted.